### PR TITLE
Documentation of some changes

### PR DESCRIPTION
**Pull request by luigifab**
Here is a draft with the changes between OM 19.x and 20.x.
It's not very good but it's a start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,15 +73,60 @@ git add -A && git commit
...
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v19.4.15...v20.0.13).
+
+### Between OpenMage 20.x and 21.x
+
+Do not use 21.x.x if you don't known what do you do, or if you don't understand consequences of the following changes.
+
+- removed mysql4 classes #1470
+- removed admin routing compatibility mode for extensions #1551
+
+Futur changes:
```
Comment (Sdfendor) [inline README.md:113]: Futur => Future

Comment (luigifab) [reply README.md]: +1

**Review by kkrieger85 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -74,15 +73,60 @@ git add -A && git commit
...
+- fixed `origData` is empty in a quote #1735
+- removed includes dir #1667
+- removed unused variable `$configSections` #1355
+- fixed incorrect datetime in customer block (`$useTimezone` parameter) #1525
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v19.4.15...v20.0.13).
+
+### Between OpenMage 20.x and 21.x
+
+Do not use 21.x.x if you don't known what do you do, or if you don't understand consequences of the following changes.
```
Comment (elidrissidev) [inline README.md:108]:
```suggestion
Do not use 21.x.x if you don't know what you're doing, or if you don't understand consequences of the following changes.
```

**Review by elidrissidev — CHANGES_REQUESTED** [review]

**Review by elidrissidev — APPROVED** [review]

**Review by Sdfendor — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -74,15 +73,40 @@ git add -A && git commit
...
+
+Do not use 20.x.x if you need IE support.
+
+- removed IE conditional comments, IE styles, IE scripts and IE eot files #1073
+- removed frontend default themes (default, modern, iphone, german, french, blank, blue) #1600
+- fixed incorrect datetime in customer block (`$useTimezone` parameter) #1525
+- add redis as a valid option for `global/session_save` #1513
+- possibility to disable global search in backend #1532
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v19.4.15...v20.0.13).
```
Comment (elidrissidev) [inline README.md:90]:
```suggestion
For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/1.9.4.x...20.0).
```

File: README.md [hunk 4]
```diff
@@ -74,15 +73,40 @@ git add -A && git commit
...
+
+### Between OpenMage 20.x and 21.x
+
+Do not use 21.x.x if you don't know what you're doing, or if you don't understand consequences of the following changes.
+
+- removed mysql4 classes #1470
+- removed admin routing compatibility mode for extensions #1551
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v20.0.13...v21.0.0).
```
Comment (elidrissidev) [inline README.md:99]:
```suggestion
For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/20.0...21.0).
```
Also, why are we documenting a version that doesn't exist yet?

**Review by elidrissidev — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (luigifab) [reply README.md:99]: Because I hope it will be released.
In UTC time there are still some hours before 2022!

Comment (elidrissidev) [reply README.md]: Well yeah, I would prefer if it aligns with year i.e 22 instead of 21

**Review by elidrissidev — DISMISSED** [review]

**Review by Sdfendor — DISMISSED** [review]

**Comment by ADDISON74** [comment]
Except the part which refers to OpenMage version 21 everything looks good to me.

**Comment by tmotyl** [comment]
I also don't know what is the story with v21?

**Comment by fballiano** [comment]
just to explain: at that time there was a discussion about the removal of the mysql classes and maybe move it a another major release called v21. then the whole thing was paused.

**Comment by luigifab** [comment]
For the History, I like my breaking ideas!

```md
### Between OpenMage 20.x and 21.x

Do not use 21.x.x if you don't know what you're doing, or if you don't understand consequences of the following changes.

- removed mysql4 classes
- removed admin routing compatibility mode for extensions

For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/20.0...21.0).
```

**Review by fballiano — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Review by elidrissidev — APPROVED** [review]

**Review by kkrieger85 — APPROVED** [review]

**Review by kiatng — APPROVED** [review]

**Comment by elidrissidev** [comment]
@fballiano Can we keep using squash and merge instead of a merge commit? It makes the history cleaner and easier to read through.

**Comment by fballiano** [comment]
> @fballiano Can we keep using squash and merge instead of a merge commit? It makes the history cleaner and easier to read through.

I'll take a look into that! Thanks for the suggestion, it actually makes a lot of sense!

**Pull request merged**